File: Maps/SumOfUniqueElement/sumOfUnique.cpp
#include <unordered_map>
#include <vector>
using namespace std;

class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        unordered_map<int,int>mp;
        int n = nums.size();
        for(int i=0;i<n;i++){
            mp[nums[i]]++;
        }
        int ans = 0;
        for(auto ele : mp)
            if(ele.second == 1) ans += ele.first;
        return ans;
    }
};

File: Maps/SumOfUniqueElement/sumOfUnique.py
from typing import List

class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        mp = {}
        n = len(nums)
        for i in range(n):
            mp[nums[i]] = mp.get(nums[i],0) + 1
        ans = 0
        for ele in mp:
            if mp[ele] == 1: ans += ele
        return ans